Packs/SophosCentral/ReleaseNotes/1_0_5.md

```diff
@@ -0,0 +1,9 @@
+
+#### Integrations
+##### Sophos Central
+- Added the *managedAgentName* and the *personName* to the following commands:
```

Comment:
What about the `Morphed`?

Reply:
not declaring them as they are a workaround and shouldn't be used.

Reply:
```suggestion
- Added the *managedAgentName* and the *personName* output paths to the following commands:
```